addons/website/views/snippets.xml

```diff
@@ -1228,11 +1228,18 @@
         data-selector="section, .parallax, .carousel-item"
         data-exclude=".s_hr, .s_image_gallery, .o_gallery .carousel-item"/>
 
-    <!-- Background Image -->
-    <div data-js="background_position"
-        data-selector="section, .parallax, .carousel-item"
-        data-exclude=".o_gallery .carousel-item">
```

Comment:
I think that the selector and exclude should not have change here. I'll change that.